doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -162,38 +119,17 @@ Forecast calibration uses the trained GBDT models, along with the forecast cube
 feature cubes. The tree-models are passed in via a model-config json which identifies
 the appropriate tree-model file for each error-threshold.
 
-Forecast calibration proceeds via a 2-step process:
+The decision-tree models are used to construct representative probability distributions for
+each input ensemble member which are then blended to give the calibrated
+distribution for the full ensemble.
 
-1. Evaluate the error CDF defined over the series of error-thresholds used in model training.
-   Each exceedance probability is evaluated using the corresponding tree-model, and the feature
-   variables as inputs.
+The distributions for the individual ensemble members are formed in a two-step process:
 
-2. Interpolate the CDF to extract a series of percentile values for the error distributions.
-   The error percentiles are then added to each associated ensemble realization from the
-   forecast variable to produce a series of realisable forecast values.
+1. Evaluate the CDF defined over the specified model thresholds for each ensemble member.
+Each threshold exceedance probability is evaluated using the corresponding
+decision-tree model.
 
-Collectively these series form the calibrated super-ensemble which is obtained by collapsing
-the two realization dimensions into one. This is then sampled to provide the calibrated
-ensemble forecast.
+2. Interpolate each ensemble member distribution to the output thresholds.
```

Comment:
The formatting in the markdown seems a bit off. Might be worth double checking what's causing the misalignment (maybe the extra line)?

Reply:
fixed, it was the extra line